content/shmem_symmetric_available.tex

```diff
@@ -0,0 +1,33 @@
+\apisummary{
+Returns the memory available on the symmetric heap in bytes.}
+
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_symmetric_available(size_t *available_size);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{OUT}{available\_size}{The symmetric heap memory available in bytes.}
+\end{apiarguments}
+
+
+\apidescription{
+The \FUNC{shmem\_symmetric\_available} call returns the memory available on the symmetric heap in bytes.
+The available memory is the unallocated memory on the symmetric heap, which could be allocated by using memory allocation routines such as \FUNC{shmem\_malloc} or \FUNC{shmem\_calloc}. 
```

Comment:
I'm not an expert in memory allocators and maybe this is something for the API notes, but I think there's a difference in the total amount of unallocated memory and the (largest) amount of memory that could be allocated by a single call to `shmem_malloc`.

Allocations and deallocations may cause some fragmentation of the symmetric heap and the size returned by this interface may be larger in cases than the largest allocation that `shmem_malloc` could actually provide.